build.sbt

```diff
@@ -15,6 +15,7 @@ libraryDependencies ++= Seq(
   "com.amazonaws" % "aws-java-sdk-cloudwatch" % awsClientVersion,
   "com.amazonaws" % "aws-java-sdk-s3" % awsClientVersion,
   "com.amazonaws" % "aws-java-sdk-sns" % awsClientVersion,
+  "com.fasterxml.jackson.core" % "jackson-core" % "2.15.4",
```

Comment:
Does this have to go in as a `dependencyOverrides` entry? Or is a straight up `libraryDependencies` inclusion enough to evict other uses everywhere? I'm not certain either way so if it looks like the build is OK like this I'm happy to approve anyway.

Reply:
Ah, ok so the [dependencyOverrides don't add a direct dependency](https://www.scala-sbt.org/1.x/docs/Library-Management.html#Overriding+a+version) but force the version to what we want. Perhaps that's more desirable than a direct dependency brought in via `libraryDependencies`?

Reply:
It’s fine as a libraryDependencies: I can see when I run `dependencyBrowseTree` that the older version is evicted for other dependencies.

Reply:
> Ah, ok so the [dependencyOverrides don't add a direct dependency](https://www.scala-sbt.org/1.x/docs/Library-Management.html#Overriding+a+version) but force the version to what we want. Perhaps that's more desirable than a direct dependency brought in via `libraryDependencies`?

Interesting, maybe that would be better: I’ll try it. I’m surprised you can override to a dependency version without explicitly depending on it, though.

Reply:
It certainly works, and means a direct dependency isn’t listed when running `dependencyBrowseTree`. I’ll go with that.